src/fileimpl.cpp

```diff
@@ -551,16 +551,36 @@ class Grouping
     struct zim_MD5_CTX md5ctx;
     zim_MD5Init(&md5ctx);
 
+    const int piece_size=1024;
```

Comment:
We tend to use `chunk_size` instead of `piece` (and even better `CHUNK_SIZE` as it is a constant)